master/internal/resourcemanagers/kubernetes_resource_manager.go

```diff
@@ -205,9 +205,15 @@ func (k *kubernetesResourceManager) receiveRequestMsg(ctx *actor.Context) error
 	case sproto.SetGroupMaxSlots:
 		k.getOrCreateGroup(ctx, msg.Handler).maxSlots = msg.MaxSlots
 
-	case sproto.SetGroupWeight, sproto.SetGroupPriority:
+	case sproto.SetGroupWeight:
 		// SetGroupWeight and SetGroupPriority are not supported by the Kubernetes RP.
 
+	case sproto.SetGroupPriority:
```

Comment:
isn't this a write side feature?

Reply:
this is necessary for determined to adjust the priority from the default to what the user sets in the config.